src/data/linux-quiz.ts

```diff
@@ -1694,6 +1694,16 @@ const linuxQuiz = [
     Distractor3: "cat /var/log/syslog",
     Explanation: "The cd command stands for change directory and is used to change to a directory in the terminal. The cd command is not used to display logs.",
     Link: "https://www.youtube.com/watch?v=ROjZy1WbCIA&t=3472s"
+  },
+  {
+    Question: "In Linux, when you remove a users account using the 'userdel' command, what happens to the data files stored in that users directory?",
```

Comment:
```suggestion
    Question: "In Linux, when you remove a user's account using the 'userdel' command, what happens to the data files stored in that user's directory?",
```

Reply:
Thank you for the feedback and I've made the changes as requested 👍